### PR TITLE
get comment from mysql

### PR DESCRIPTION
**Pull request by sonkyokukou**

### CODE REVIEW

**Comment by sonkyokukou** [comment]
get field comment from mysql

File: src/Xethron/MigrationsGenerator/Generators/FieldGenerator.php [hunk 1]
```diff
@@ -146,6 +191,7 @@ protected function getFields($columns, IndexGenerator $indexGenerator)
 			if ($nullable) $decorators[] = 'nullable';
 			if ($default !== null) $decorators[] = $this->getDefault($default, $type);
 			if ($index) $decorators[] = $this->decorate($index->type, $index->name);
+			if ($this->defaultConnection=='mysql' && $comment) $decorators[] = "comment('{$comment}')";
```
Comment (jamesalton) [inline src/Xethron/MigrationsGenerator/Generators/FieldGenerator.php]: A single quote in the existing comment field will mess up here unless you escape $comment first.
```
                        if ($this->defaultConnection=='mysql' && $comment) {
                                $escapedComment = DB::connection()->getPdo()->quote($comment);
                                $decorators[] = "comment({$escapedComment})";
                        }
```

**Comment by Xethron** [comment]
Hello @sonkyokukou, thank you so much for submitting this. And sorry for only getting back to you now.

I believe #61 is a similar implementation? But it seems a lot simpler... Is this true? Or am I missing something?

In other words, is there something yours does that might not work in #61?

**Comment by Xethron** [comment]
This one doesn't seem to work as expected. It creates a field: `$table->comment='Table Comment';` with the correct comment yes, however, that field doesn't migrate to the database. And from what I can see, Laravel doesn't support table comments at all. Only field comments...

**Comment by Xethron** [comment]
Closing this for now. You are most welcome to re-open if the #61 implementation doesn't work for you :) Thanks for taking the time to submit this.

**Pull request closed without merging**